src/open-websocket.js

```diff
@@ -1,14 +1,15 @@
 import ws from 'isomorphic-ws'
 import ReconnectingWebSocket from 'reconnecting-websocket'
 
-export default url => {
+export default (url,proxyOpt) => {
   const rws = new ReconnectingWebSocket(url, [], {
     WebSocket: ws,
     connectionTimeout: 4e3,
     debug: false,
     maxReconnectionDelay: 10e3,
     maxRetries: Infinity,
     minReconnectionDelay: 4e3,
+    ...proxyOpt
```

Comment:
I dont see an option on reconnecting-websocket for a proxy, are you sure about this?